doc/deployment/migrations.md

```diff
@@ -1,45 +1,41 @@
-
-## Migrate to 1.5.x
-
-To migrate from 1.4.x to 1.5.x, use the `pachctl migrate` command.  See `pachctl migrate --help` for detailed instructions.
-
-As an example, to migrate from 1.4.8 to 1.5.0, use the following command:
-
-```
-$ pachctl migrate --from 1.4.8 --to 1.5.0
-```
-
-Note that the `pachctl migrate` command can be run either before or after you've redeployed your cluster with the new version (e.g. via `pachctl deploy`).
-
-Most importantly, you need to ensure that your cluster is "at rest" when you run `pachctl migrate`.  That is, there shouldn't be any ongoing activities that are changing the state of the cluster.  Examples would be running jobs or ongoing `put-file` requests.
-
-*Note: For v1.4 pipelines that specify environment variables in their pipeline specs, you will unfortunately need to reprocess all data for those pipelines as part of the v1.5 migration. This will automatically happen as part of the first job that spawns after the migration. Sorry for inconvenience.* 
-
-## Backup
-
-It’s paramount that you backup your data before running a migration.  While we’ve tested the migration code extensively, it’s still possible that they contain bugs, or that you accidentally use them in a wrong way.
-
-In general, there are two data storage systems that you might consider backing up: the metadata storage and the data storage. Not all migration scripts touch both systems, so you might only need to back up one of them. Look at the README for a particular migration script for details.
-
-### Backup the metadata store
-
-Assuming you’ve deployed Pachyderm on a public cloud, your metadata is probably stored on a persistent volume. See the respective [Deploying Pachyderm](http://pachyderm.readthedocs.io/en/stable/deployment/deploy_intro.html) guide for details.
-
-Here are official guides on backing up persistent volumes for each cloud provider:
-
-- [GCE Persistent Volume](https://cloud.google.com/compute/docs/disks/create-snapshots)
-- [Elastic Block Store (EBS)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-creating-snapshot.html)
-
-### Backup the object store 
-
-We don’t currently have migration scripts that affect the object store.
-
+New versions of Pachyderm often require a migration for some or all of the on
+disk objects. This document describes how Pachyderm migration works and
+the best practices surrounding it.
+
+As of 1.7, Pachyderm's migration works by extracting objects into a stream of
+API requests, and replaying those requests onto the newer version of pachd.
+This process happens automatically using Kubernetes' "rolling update"
+functionality. All you need to do is deploy Pachyderm (with `pachctl deploy) as
+you would if you were deploying for the first time. While migration is
```

Comment:
I think this is a little confusing. Does this mean that:

1. I have a previous version of Pachyderm up and running.
2. I update `pachctl` to the new version.
3. I run my `pachctl deploy ...` command with the updated `pachctl`, thus deploying the new version side-by-side with the old version.

If so, it might be worth specifying that as steps, similar to above. Because some people won't get that from this quick statement I think.